### ISSUE TITLE
class labels for the testing data set

### ISSUE THREAD

**Issue by ra9118** [hil-se/fds#34]
Can we have the class label for the testing set to measure the accuracy of our model?
The current testing set doesn't have the class label. I tried to measure the accuracy by dividing the training set into training and testing but the samples is not enough . I t will be good to have class label the current testing set, such as  having values at class label  "Species" for Iris_test.csv

**Comment by azhe825** [comment]
That testing set does not have labels. You can compare your results with sklearn results for correctness. No need for actual labels.

**Comment by ra9118** [comment]
got it. Thanks, Professor
